guidelines/index.html

```diff
@@ -1170,6 +1177,15 @@
 					<p>audio description that is added to an audiovisual presentation by pausing the <a>video</a> so that there is time to add additional description</p>
 					<p>This technique is only used when the sense of the <a>video</a> would be lost without the additional <a>audio description</a> and the pauses between dialogue/narration are too short.</p>
 				</dd>
+        <dl>
+		          <dt><dfn>easily available (or easily available mode or setting) one or more of the following are true:</dfn></dt>
+                  <dd>
+        <ul>
+  <li>can be set one time with as wide a scope as possible (such as using the standards of the OS, From ISO 9241-112 or GPII when available) </li>
+  <li>has the option to save or change the setting, where available interoperably, but also for the scope of the set of web pages </li>
+  <li>is reachable from each screen where it may be needed, and the path and the control conforms to all of this document</li>
```

Comment:
* Replace "screen" with "webpage", which has an existing glossary definition
* Delete "and the path and control conforms to all of this document".  This is not only implied in my opinion, but also does not specify a conformance level and conflicts/overlaps with Note 2 in the [definition of mechanism](https://www.w3.org/TR/WCAG20/#mechanismdef).
* "Reachable" is not defined and so this requirement is very vague.  Technically, if I need to click Settings and then jump through 10 pages of controls to get to the mechanism for suppressing the interruption, that is reachable.

If the intent here is to be able to suppress with a single click, then maybe something like: "able to be set without any change of content".